internal/providers/terraform/azure/application_insights_web_t.go

```diff
@@ -1,51 +1,27 @@
 package azure
 
 import (
-	"strings"
-
-	"github.com/shopspring/decimal"
-	"github.com/tidwall/gjson"
-
+	"github.com/infracost/infracost/internal/resources/azure"
 	"github.com/infracost/infracost/internal/schema"
 )
 
-func GetAzureRMApplicationInsightsWebRegistryItem() *schema.RegistryItem {
+func getAzureRMApplicationInsightsWebRegistryItem() *schema.RegistryItem {
```

Comment:
```suggestion
func getApplicationInsightsWebTestRegistryItem() *schema.RegistryItem {
```
I assume the filename is `_t.go` because of the `_test.go` special case.